Jenkinsfile_nightly

```diff
@@ -59,27 +59,17 @@ withNightlyPipeline("nodejs", product, component) {
     env.TEST_E2E_WAIT_FOR_ANGULAR = 'false'
     env.TEST_E2E_FAIL_FAST = 'false'
     env.TEST_E2E_ANNOTATION = params.NIGHTLY_TAG
-    enableFullFunctionalTest(120)
+    enableCrossBrowserTest()
 
```

Comment:
The will enable only one scenario with @xbrowsertest and also please increase the pipeline time by adding (120)

Reply:
> The will enable only one scenario with @xbrowsertest and also please increase the pipeline time by adding (120)

This has been addressed as part of https://github.com/hmcts/ia-ccd-e2e-tests/pull/409/